### PR TITLE
SorrTask489-Convert-Hubstaff-gDoc-to-Markdown

### PR DESCRIPTION
**Pull request by gurtajbir**
#489 
Closed previous [PR](https://github.com/sorrentum/sorrentum/pull/531)

### CODE REVIEW

File: dev_scripts/convert_docx_to_markdown.py.log
Comment (DanilYachmenev) [inline dev_scripts/convert_docx_to_markdown.py.log:1]: no need for the log to be commited

File: docs/Hubstaff.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+- time tracking
+
+- invoice creation
+
+- payment
+
+The goal is to replace our
+[<span class="underline">hour log spreadsheet</span>](https://docs.google.com/spreadsheets/d/1oNd6ORhc94oUzg5nhNC7fQelN_PmfAv110F7lUiZsxo/edit#gid=0)
+with Hubstaff and get paid automatically for the worked hours
```
Comment (DanilYachmenev) [inline docs/Hubstaff.md:27]: put all the text in bullet point, just check the indents
```
- [<span class="underline">Hubstaff</span>](https://hubstaff.com/) is a tool for
remote working that automates:
    - time tracking
    - invoice creation
    - payment
- The goal is to replace our
[<span class="underline">hour log spreadsheet</span>](https://docs.google.com/spreadsheets/d/1oNd6ORhc94oUzg5nhNC7fQelN_PmfAv110F7lUiZsxo/edit#gid=0)
with Hubstaff and get paid automatically for the worked hours
```

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt [hunk 2]
```diff
@@ -1 +1 @@
-abc
```
Comment (DanilYachmenev) [inline helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: such files should not be committed
remove them by running `amp/dev_scripts/clean_up_text_files.sh` on them in `Test_on_file` mode (or add end of line char at the end of them)
same for all the test outcomes updates here
only doc related files should be commited

**Review by DanilYachmenev — COMMENTED** [review]
see the comments

File: dev_scripts/convert_docx_to_markdown.py.log
Comment (gurtajbir) [reply dev_scripts/convert_docx_to_markdown.py.log:1]: Removed

File: docs/Hubstaff.md (hunk 1 above)
Comment (gurtajbir) [reply docs/Hubstaff.md]: Done

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt (hunk 2 above)
Comment (gurtajbir) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: Just manually added `\n` at the end

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt [hunk 3]
```diff
@@ -1 +1 @@
...
+abc\n
```
Comment (DanilYachmenev) [inline helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: well, you can see that it actually did not work)
so lets remove the addition and try to get rid of these outcomes with the mentioned script

Comment (gurtajbir) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: quick question on the usage of [dev_scripts](https://github.com/sorrentum/sorrentum/tree/master/dev_scripts)
/clean_up_text_files.sh]. I edited the MODE and set it to `Test_on_file`.  I am not sure about how to specify the file to be acted on. Currently, I am doing `dev_scripts/clean_up_text_files.sh FILE="helpers/test/outcomes/TestCheckString1.test_check_string1/output/test.txt"`. It does not seem to do anything.

Comment (DanilYachmenev) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: Well, it seems that you're doing all correctly
You need to set `FILE=` under `if [[ $mode == "Test_on_file" ]]; then` clause line and then run the `.sh` script from bash
Снимок экрана 2023-08-23 в 02 44 10

file should appear in the changed files section in git

**Comment by DanilYachmenev** [comment]
obsolete, closing

**Pull request closed without merging**